### PR TITLE
🌐 sv_SE | Small fixes | missionhelper

### PR DESCRIPTION
**Pull request by dennziil**

### CODE REVIEW

**Comment by Suchty112** [comment]
You commits https://github.com/LSS-Manager/LSSM-V.4/pull/355/commits/7c0da8eee7ab514d2fa89a3250f3194f4ea1bc5f and
https://github.com/LSS-Manager/LSSM-V.4/pull/355/commits/441d7ae2c887f4f458ce13aa534723e207acbc21 were not correct. in the main file credits and coins must remain so. mostly only the text in the "" or '' must be changed.

File: src/i18n/sv_SE.ts [hunk 1]
```diff
@@ -808,7 +808,7 @@ export default {
     alliance: 'Allians',
     premiumNotice:
         'Den här funktionen utökar en premiumfunktion i spelet och är därför endast tillgänglig för spelare med ett Larmcentralen-spelet premiumkonto!',
-    credits: 'Krediter',
+    credits: 'credits',
```
Comment (JRH-1997) [inline src/i18n/sv_SE.ts:811]: @Suchty112 Why did you change this translation? That one is correct when it's in sv_SE

Comment (dennziil) [reply src/i18n/sv_SE.ts:811]: Whoops. fixing!

Comment (Suchty112) [reply src/i18n/sv_SE.ts:811]: "credits: 'Krediter'," that is, of course, correct. 
Sorry, my programm has renamed too much

**Review by Suchty112 — APPROVED** [review]
